### DISCUSSION TITLE
Multiselect search input access

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#3473] [Q&A] [answered]
Hi.
I have a wrapper around any input type in mantine lib to have floating labels effect (like in material ui).
So, with multiselect I have a problem, because we have an advanced search input inside (when searchable).
So, I want to have access to it to handle onFocus, and onBlur events.
How to do this?
Thanks.

**Comment by rtivital** [comment, marked as answer]
Set `onFocus` and `onBlur` props, is there any issue with that?
https://codesandbox.io/s/blissful-satoshi-2r1h5s?file=/src/App.tsx
